### ISSUE TITLE
Hooks

### ISSUE THREAD

**Issue by reiz** [versioneye/dependency_wheel#5]
Currently the "show_info_box()" method is hard wired into the code in the dependencywheel.js file. Would be cool if a method name could be give as a parameter. In that way it would be a hook.

**Comment by coding46** [comment]
Maybe we should pass it as method name or as callback method in the options hash.

Do we need a fallback if none of both is set?

**Comment by reiz** [comment]
Yes. That's a good idea! I think a callback would be the best solution.
